guides/source/active_record_multiple_databases.md

```diff
@@ -203,7 +203,7 @@ database you can run `bin/rails db:create:animals`.
 ## Connecting to Databases without Managing Schema and Migrations
 
 If you would like to connect to an external database without any database
-management tasks such as schema management, migrations, seeds, etc. you can set
+management tasks such as schema management, migrations, seeds, etc. You can set
```

Comment:
The `you` is correct in the original because it's a continuation of the sentence. Instead there's a missing comma according to https://english.stackexchange.com/questions/283086/when-putting-etc-in-the-middle-of-a-sentence-do-you-use-etc-or-etc-or

```suggestion
management tasks such as schema management, migrations, seeds, etc., you can set
```

Reply:
Change done.

Thank you.